modules/dxp/apps/app-builder-workflow/app-builder-workflow-web/src/main/resources/META-INF/resources/js/pages/entry/ViewEntry.es.js

```diff
@@ -139,6 +163,7 @@ export default function ViewEntry({
 			<ViewEntryUpperToolbar
 				dataRecordId={dataRecordId}
 				page={page}
+				showButtonsOptions={showButtonsOptions}
```

Comment:
@naraandrade could you update this property name for "showOptions", as you were change at before commit of "ViewEntryUpperToolbar.es.js" file? Thanks!